### PR TITLE
Feature/#175

### PR DESCRIPTION
**Pull request by GalochkinKirov**
https://github.com/OAGr/foretold/issues/175

### CODE REVIEW

File: bots/bots/aggregation-bot.js [hunk 1]
```diff
@@ -13,29 +17,53 @@ class AggregationBot {
...
+    }).map((measurement) => {
+      const xs = _.get(measurement, 'value.floatCdf.xs');
+      const ys = _.get(measurement, 'value.floatCdf.ys');
+      return new Cdf(xs, ys);
+    });
+
+    if (cdfs.length === 0) return null;
+
+    // @todo: What's about the limit?
+    const combined = new CdfCombination(cdfs).combine(10);
```
Comment (OAGr) [inline bots/bots/aggregation-bot.js]: The sample count should be 1000, but should probably also be a constant on the top of the file or config or similar

File: bots/bots/aggregation-bot.js [hunk 2]
```diff
@@ -13,29 +17,53 @@ class AggregationBot {
...
+      if (!aggregated) continue;
+
+      await this.api.measurementCreateAggregation({ ...id, ...aggregated });
     }
 
     return true;
   }
 
   /**
+   * @todo: What to do with "floatPoint" spot?
```
Comment (OAGr) [inline bots/bots/aggregation-bot.js]: points should never be aggregated. Points can only be final values, not competitive measuremenets.

File: bots/bots/aggregation-bot.js (hunk 1 above)
Comment (GalochkinKirov) [reply bots/bots/aggregation-bot.js]: Ok, will update

File: bots/bots/aggregation-bot.js (hunk 2 above)
Comment (GalochkinKirov) [reply bots/bots/aggregation-bot.js]: ok

File: bots/bots/aggregation-bot.js [hunk 3]
```diff
@@ -13,29 +17,53 @@ class AggregationBot {
    */
   async main() {
     const measurables = await this.api.measurables();
```
Comment (OAGr) [inline bots/bots/aggregation-bot.js]: It's important to only get measurables that have competitive entrees that are newer than aggregates by this specific bot. Can that be added here, or with new stories?

Comment (GalochkinKirov) [reply bots/bots/aggregation-bot.js]: I think would be better in another issue

Comment (OAGr) [reply bots/bots/aggregation-bot.js]: ok, plz review this new issue.
https://github.com/OAGr/foretold/issues/187

File: bots/bots/aggregation-bot.js (hunk 1 above)
Comment (GalochkinKirov) [reply bots/bots/aggregation-bot.js]: Fixed.

**Pull request merged**